chopper/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.0.9
```

Comment:
Can you please remove this entry? I'll add it later once I make the actual release.

Reply:
Remove all the 7.0.9 section or only the line 3